### PR TITLE
Bug 1217734 - Deleted duplicate home screens can be selected

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5b7d17594c68e6e7d323ae00a90438c90c9dd6ff

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=796581c7ddc9e5971aa7b443aacab71164048aea

File: apps/settings/js/modules/apps_cache.js [hunk 1]
```diff
@@ -79,18 +79,18 @@ define(function(require) {
 
       mozAppsMgmt.onuninstall = function(evt) {
         var manifestURL = evt.application.manifestURL;
-        var removedAppIndex;
         // installed apps are normally in the end, so let's iterate
```
Comment (gasolin) [inline apps/settings/js/modules/apps_cache.js]: the comments are incorrect now, please update them

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b2f1c824e753cfb6cfbc26f74f15fceabeb5f943

**Pull request merged**